From 43dda7dbe5ec831e2f0f14626e707200dd421239 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Fri, 16 Oct 2020 19:11:47 -0700
Subject: [PATCH 3/7] Cite Tom Scott on Danish time

* tz-art.html (Documentaries): Add "Why Denmark is .17 Seconds
Behind The World" (thanks to Jim Eggert).
---
 tz-art.html | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tz-art.html b/tz-art.html
index 5de546a..72b9654 100644
--- a/tz-art.html
+++ b/tz-art.html
@@ -22,6 +22,10 @@ into problems that programmers have with timekeeping.</li>
 Explaining The Mysteries Of Time Zones</a>" (2017; 2:15)
 briefly says why France has more time zones than Russia.
 <li>
+"<a href="https://www.youtube.com/watch?v=yRz-Dl60Lfc">Why Denmark is
+.17 Seconds Behind The World</a>" (2019; 6:29) explains why Denmark and
+the United Kingdom don't exactly follow their own law about civil time.
+<li>
 "About Time" (1962; 59 minutes) is part of the
 Bell Science extravaganza, with Frank Baxter, Richard Deacon, and Les Tremayne.
 Its advisor was Richard Feynman, and it was voiced by Mel Blanc.
-- 
1.8.3.1

